recipe/bld.bat

```diff
@@ -25,7 +25,7 @@ if errorlevel 1 (
    appveyor PushArtifact "%CD%\config.log"
    exit 1
 )
-make
+make -j%NUMBER_OF_PROCESSORS%
```

Comment:
Same comment from before: 

Better to avoid -j. CIs do not always report it reliably. Instead, prefer to set MAKEFLAGS and let conda build pass that along.